server/controllers/admin.py

```diff
@@ -29,6 +29,7 @@
 import server.jobs.example as example
 import server.jobs.moss as moss
 import server.jobs.github_search as github_search
+import server.jobs.export as export
```

Comment:
Nit: I think we can combine these four lines as
`from server.jobs import example, export, github_search, moss`